pipeline.py

```diff
@@ -112,32 +91,32 @@ def pipeline(
             int
         ] = None,  # FIXME: https://issues.redhat.com/browse/RHOAIRFE-467
         sdg_base_model: str = "s3://<BUCKET>/<PATH_TO_MODEL>",
-        sdg_scale_factor: int = 2,  # Renamed upstream https://github.com/instructlab/instructlab/blob/f7d40f6ed5112d59132dd832bd332fa6fbbe7010/src/instructlab/configuration.py#L279-L290
-        sdg_pipeline: str = SDG_PIPELINE,
-        sdg_max_batch_len: int = MAX_BATCH_LEN,
-        sdg_sample_size: float = 1.0,
+        sdg_scale_factor: int = 30,  # https://github.com/instructlab/instructlab/blob/d68c5a12026e08092bd4171a77667b09f7c36f72/tests/testdata/default_config.yaml#L125
+        sdg_pipeline: str = "full",  # https://github.com/instructlab/instructlab/blob/d68c5a12026e08092bd4171a77667b09f7c36f72/tests/testdata/default_config.yaml#L122
+        sdg_max_batch_len: int = 5000,  # https://github.com/instructlab/instructlab/blob/d68c5a12026e08092bd4171a77667b09f7c36f72/tests/testdata/default_config.yaml#L334
+        sdg_sample_size: float = 1.0,  # FIXME: Not present in default config. Need justification?
```

Comment:
You can point to https://github.com/instructlab/sdg/pull/432 for further details.

Reply:
Sure I can. However I don't understand it 😄 Can you please help me understand how that PR explains that `1.0` is a good default value? The PR doesn't even mention `1.0` being a valid value...

Reply:
Yes, sounds good, thanks!

Reply:
Thank you for having the link at your disposal. 🙂 Really. 🙂  It's gonna make the handover so much less painful. 🙌